composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "faktiva/prestashop-seo-tk",
+    "name": "faktiva/faktiva_seo_tk",
```

Comment:
I think this change is not needed: https://github.com/composer/installers#custom-install-names